File: Practice(Beginner)/Python3/Chef_and_Cook-Off.py
# Source: https://www.codechef.com/problems/CCOOK

t = int(input())
for test in range(t):
    l = list(map(int, input().split()))
    c = l.count(1)
    if(c == 0):
        print("Beginner")
    elif(c == 1):
        print("Junior Developer")
    elif(c == 2):
        print("Middle Developer")
    elif(c == 3):
        print("Senior Developer")
    elif(c == 4):
        print("Hacker")
    elif(c == 5):
        print("Jeff Dean")

File: Practice(Beginner)/C++/ Chef and Cook-Off.cpp
// Source: https://www.codechef.com/problems/CCOOK

#include <bits/stdc++.h>
using namespace std;
int main(){
    int t, temp;
    cin >> t;
    while (t--){
        int c = 0;
        for (int i = 0; i < 5; ++i) {
            cin >> temp;
            if(temp == 1){
                c++;
            }
        }
        if(c == 0){
            cout << "Beginner" << endl;
        } else if (c == 1){
            cout << "Junior Developer" << endl;
        } else if (c == 2){
            cout << "Middle Developer" << endl;
        } else if (c == 3){
            cout << "Senior Developer" << endl;
        } else if (c == 4){
            cout << "Hacker" << endl;
        } else if (c == 5){
            cout << "Jeff Dean" << endl;
        }
    }
}